mlflow/server/validation.py

```diff
@@ -0,0 +1,35 @@
+import re
+from typing import List, Union
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+
+VALID_CONTENT_TYPE_CHARS = re.compile(r"^[a-zA-Z0-9\-/;,+_ ]*$")
```

Comment:
```suggestion
VALID_CONTENT_TYPE_CHARS = re.compile(r"^[a-zA-Z0-9\-/;,+_ ]")
```

Do we need `*$"`?

Reply:
We need `*$" and ^`. I thought this is a set of invalid characters.

Reply:
Yup, let me rename it to sth like `VALID_CONTENT_TYPE_CHAR_PATTERN` to avoid confusion:)